srcpkgs/qsudo/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'qsudo'
+pkgname=qsudo
+version=2019.11.29
+revision=1
+build_wrksrc="src-qt5"
+build_style=qmake
+hostmakedepends="qt5-host-tools"
+makedepends="qt5-devel"
+depends="qt5 sudo"
```

Comment:
Adding qt5 here is not necessary, it is automatically detected via shlibs

Reply:
Removed!